src/notebookConverter.ts

```diff
@@ -170,21 +171,21 @@ export class NotebookConverter implements Disposable {
     }
 
     // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
-    public toIncomingWorkspaceSymbols(symbols: SymbolInformation[] | null | undefined) {
+    public toNotebookWorkspaceSymbols(symbols: SymbolInformation[] | null | undefined) {
         if (Array.isArray(symbols)) {
-            return symbols.map(this.toIncomingWorkspaceSymbol.bind(this));
+            return symbols.map(this.toNotebookWorkspaceSymbol.bind(this));
         }
         return symbols ?? undefined;
     }
 
-    public toIncomingWorkspaceEdit(workspaceEdit: WorkspaceEdit | null | undefined): WorkspaceEdit | undefined {
```

Comment:
A lot of the change is changing 'toIncoming' to 'toNotebook' and 'toOutgoing' to 'toConcat'. I kept forgetting which was 'outgoing' and which was 'incoming'

Reply:
Smart, feels much clearer to me this way.